pywinauto/base_application.py

```diff
@@ -594,13 +595,19 @@ def _ctrl_identifiers(self):
 
         return control_name_map
 
-    def print_control_identifiers(self, depth=None, filename=None):
+    def print_control_identifiers(self, depth=30, max_width=5, filename=None):
         """
         Prints the 'identifiers'
 
         Prints identifiers for the control and for its descendants to
         a depth of **depth** (the whole subtree if **None**).
 
+        :param depth: Max depth level of an element tree to print (sNone - infinite).
+
+        :param max_width: Max amount of children elements of each node to print (None - infinite)
```

Comment:
-> `Max number of children of each element to dump`

Reply:
fixed